examples/openIMIS/test_entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+docker-compose build db backend
+docker-compose up -d db backend
+
+docker-compose exec backend chmod +x /openimis-be/openIMIS/custom_scripts/check_service_availability.sh
```

Comment:
you can copy `check_service_availability.sh` script to main folder instead of openIMIS to simplify path